cla-backend-go/v2/gitlab-activity/models/models.go

```diff
@@ -0,0 +1,17 @@
+// Copyright The Linux Foundation and each contributor to CommunityBridge.
+// SPDX-License-Identifier: MIT
+
+package models
```

Comment:
Maintain convention with models.py under signature package

Reply:
I have updated this , moved to gitlab_activity package.